meta/system.py

```diff
@@ -487,6 +603,16 @@ def align(self, name, corpus, flow, feature_scorer, **kwargs):
     def estimate_mixtures(
         self, name, corpus, flow, old_mixtures=None, alignment=None, prefix="", **kwargs
     ):
+        """
+        :param str name:
+        :param str corpus:
+        :param str flow:
+        :param str|list[str]|tuple[str]|rasr.FlagDependentFlowAttribute old_mixtures:
```

Comment:
Sure that `rasr.FlagDependentFlowAttribute` describes a mixture file?

Reply:
You are right. It is not. It should be Sisyphus Path object